ids/short_node_id.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package ids
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/staking"
+	"github.com/ava-labs/avalanchego/utils"
+	"github.com/ava-labs/avalanchego/utils/hashing"
+)
+
+const (
+	ShortNodeIDPrefix = "NodeID-"
+	ShortNodeIDLen    = ShortIDLen
+)
+
+var (
+	EmptyShortNodeID = ShortNodeID{}
+
+	errShortNodeID = errors.New("insufficient ShortNodeID length")
+
+	_ utils.Sortable[ShortNodeID] = ShortNodeID{}
+)
+
+type ShortNodeID ShortID
+
+// WritableShortNode is an helper which helps modifiying NodeID content
+func WritableShortNode(id *ShortNodeID) []byte {
+	return id[:]
+}
```

Comment:
I feel like we can probably just directly reference the bytes with `[:]` rather than using this

Reply:
It's not even used, leftover from a cleanup. Dropped